packages/jaeger-ui/src/components/App/TopNav.tsx

```diff
@@ -96,9 +96,14 @@ function getItem(item: ConfigMenuItem) {
 function CustomNavDropdown({ label, items }: ConfigMenuGroup) {
   const menuItems = <Menu>{items.map(getItem)}</Menu>;
   return (
-    <Dropdown overlay={menuItems} placement="bottomRight">
-      <a>
-        {label} <DownOutlined />
+    <Dropdown
+      trigger={['click']}
```

Comment:
Previously, "About Jaeger" dropdown was used to trigger automatically on hover. This attribute changes that behavior to trigger on click.

Reply:
Why do we want to change it like that?

Reply:
Should I revert this `trigger` change? I think we had a conversation before about this behavior.

> The About Jaeger menu also self-activates on hover.
https://github.com/jaegertracing/jaeger-ui/pull/1748#discussion_r1313883548

Reply:
We should always try to keep the existing behavior. It's not just the menu, the trace view selector also activates on hover, so it's a pattern.

Reply:
Got it 🚀